### PR TITLE
Blog for Photo Sharing App

### PR DESCRIPTION
**Pull request by Aneesh02**
This is my final blog.

### CODE REVIEW

**Comment by KShivendu** [comment]
Can you please add a few screenshots from the demo?

File: content/blog/aneesh.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: My experience with FOSS Overflow
+author: Aneesh Tickoo
+date: '2022-01-21'
+---
+
+# Introduction
```
Comment (KShivendu) [inline content/blog/aneesh.md:7]: `## Introduction`

File: content/blog/aneesh.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Achievements during FOSSOverflow'21
+Some of my achievements during the project:🦾
+
+1. Developing a basic UI for users
+2. Allowing users of a single room to upload their photos
+3. Implementing OpenCV's face recognition library to encode faces
+4. Using DBSCAN to cluster faces together
+5. Finally, showing face albums in order of their similarities to each other
+
+and much more...
```
Comment (KShivendu) [inline content/blog/aneesh.md:35]: Add a few screenshots and some technical details about how the project works. What are the different steps that you took while grouping the photos? (jut an overview)

File: content/blog/aneesh.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: My experience with FOSS Overflow
```
Comment (KShivendu) [inline content/blog/aneesh.md:2]: You may try to come up with a different title.

**Comment by Aneesh02** [comment]
@KShivendu Please have a look now. Thanks!

**Pull request closed without merging**